modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/validation/ParameterTypeModelValidator.java

```diff
@@ -132,6 +146,17 @@ private void checkInvalidFieldAnnotations(NamedObject model, String typeName, Fi
           }
         }
       }
+
+      private boolean isPojoWithDefaultConstructor(Class<?> clazz) {
+        return (!clazz.isInterface() && !isAbstract(clazz.getModifiers())
+            && Stream.of(clazz.getConstructors())
+                .noneMatch(c -> c.getParameterCount() == 0));
+      }
+
+      private Class<?> getTypedValueGenericClass(Type type) {
+        return type.getGenerics().size() != 0 ? type.getGenerics().get(0).getConcreteType().getDeclaringClass().get()
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Call "Optional#isPresent()" before accessing the value. ([squid:S3655](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS3655))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7615&issues=AWhiR_G7otIXnmsUW4nM&open=AWhiR_G7otIXnmsUW4nM)

[ISSUE_KEY]: AWhiR_G7otIXnmsUW4nM